From a48d6602c80f0a29d07475fccf9debcaaf005ca0 Mon Sep 17 00:00:00 2001
Message-Id: <a48d6602c80f0a29d07475fccf9debcaaf005ca0.1344844502.git.minovotn@redhat.com>
In-Reply-To: <0c740d356f8d30310bcff2b06cfefad0e72769e2.1344844502.git.minovotn@redhat.com>
References: <0c740d356f8d30310bcff2b06cfefad0e72769e2.1344844502.git.minovotn@redhat.com>
From: Alex Williamson <alex.williamson@redhat.com>
Date: Tue, 7 Aug 2012 19:41:27 +0200
Subject: [PATCH 3/5] acpi_piix4: Remove PCI_RMV_BASE write code

RH-Author: Alex Williamson <alex.williamson@redhat.com>
Message-id: <20120807194126.6021.48081.stgit@bling.home>
Patchwork-id: 40607
O-Subject: [RHEL6.4 qemu-kvm PATCH 3/5] acpi_piix4: Remove PCI_RMV_BASE write code
Bugzilla: 807391
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>

Bugzilla: 807391
Upstream commit: 31745aabcd6dce5583dbd0e5ddee93ff9fdfe3e6

Clarify this register as read-only and remove write code.  No
change in existing behavior.

Signed-off-by: Alex Williamson <alex.williamson@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
---

 hw/acpi.c |    6 ------
 1 file changed, 6 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/acpi.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/hw/acpi.c b/hw/acpi.c
index 54d4d91..6db0693 100644
--- a/hw/acpi.c
+++ b/hw/acpi.c
@@ -886,11 +886,6 @@ static uint32_t pcirmv_read(void *opaque, uint32_t addr)
     return s->pci0_hotplug_enable;
 }
 
-static void pcirmv_write(void *opaque, uint32_t addr, uint32_t val)
-{
-    return;
-}
-
 static const char *model;
 
 static int piix4_device_hotplug(PCIDevice *dev, int state);
@@ -916,7 +911,6 @@ void piix4_acpi_system_hot_add_init(PCIBus *bus, const char *cpu_model)
     register_ioport_write(PCI_EJ_BASE, 4, 4, pciej_write, pm_state);
     register_ioport_read(PCI_EJ_BASE, 4, 4,  pciej_read, pm_state);
 
-    register_ioport_write(PCI_RMV_BASE, 4, 4, pcirmv_write, pm_state);
     register_ioport_read(PCI_RMV_BASE, 4, 4,  pcirmv_read, pm_state);
 
     model = cpu_model;
-- 
1.7.11.2

